templates/rhpam713-authoring-ha.yaml

```diff
@@ -421,15 +421,6 @@ parameters:
     name: AUTH_LDAP_BIND_CREDENTIAL
     example: "Password"
     required: false
-  - displayName: LDAP Flag Login Module
-    name: AUTH_LDAP_LOGIN_MODULE
-    description: "A flag to set login module to optional. The default value is required"
-    example: "optional"
-    required: false
```

Comment:
@spolti is possible to keep this flag in templates/images and have fallback method for LDAP login module? We should keep this functionality in images if it is possible